src/partials/head-meta.hbs

```diff
@@ -1,5 +1,7 @@
     {{!-- Add additional meta tags here --}}
+    {{#if (eq "labs" page.attributes.theme)}}
     <meta name="type" content="{{{ or page.attributes.type 'Developer Guide' }}}">
+    {{/if}}
```

Comment:
suggestion: we should probably add a comment to explain why this meta tag is only enabled on "labs" pages

Reply:
I don't know what its purpose is...

Reply:
... I mean: should I be specifying it in the docs playbook? Why do we default to 'Developer Guide'?

Reply:
I guess first come, first served!
I would prefer to keep the value until we know for sure if it's used or not.

You can declare `:page-type: Reference Docs`

Reply:
> I guess first come, first served!

Fair enough!

I've added `:page-type: Docs` to our playbook for when its needed.